core-bundle/src/Resources/contao/languages/en/tl_image_size.xlf

```diff
@@ -50,6 +50,12 @@
       <trans-unit id="tl_image_size.cssClass.1">
         <source>The CSS classes will be applied to the image tag.</source>
       </trans-unit>
+      <trans-unit id="tl_image_size.skipIfDimensionsMatch.0">
+        <source>Skip if dimensions match</source>
+      </trans-unit>
+      <trans-unit id="tl_image_size.skipIfDimensionsMatch.1">
+        <source>Skip image processing if no resize is needed and use the original image file instead.</source>
```

Comment:
I would add a small comment on why it would make sense not to enable this. I guess many people would not know. Something like `If processing is skipped, no optimizations such as meta data stripping or auto rotation is applied` or maybe you find something better here.

Reply:
Added in 5bbc7c82887836a184fb14c32ab497e3e25cab45
Side note: auto rotation is always applied, regardless of this checkbox